mne/io/snirf/tests/test_snirf.py

```diff
@@ -89,6 +90,44 @@ def test_snirf_against_nirx():
     assert_allclose(raw.get_data(), raw_orig.get_data())
 
 
+@requires_h5py
+@requires_testing_data
+def test_snirf_nonstandard(tmpdir):
```

Comment:
Ok I have tried to manually hit some additional lines to increase the coverage. But I can't think of a nice way to hit the dig points without lots of manual code.

Reply:
Hah! I considered suggesting that you mock/create the fields by copying and modifying, but didn't want to make a bunch of extra work for you. But you did it anyway. :)

This seems fine to me, especially if there are complementary tests in `mne-nirs` for round-trip-ness.

Perhaps at some point we can convince @agramfort to include the writer in MNE-Python by doing a comparison of the number of lines required to mock the fields vs lines required to just include the writer :)

Reply:
Great. Yeah I should have just done this from the start, was being lazy.

> Perhaps at some point we can convince agramfort to include the writer in MNE-Python

I'm ok with maintaining things in MNE-NIRS, particularly since you both kindly review things there on request too. In my mind I am starting to have a clear line for what's appropriate where. Roughly epoching style analysis goes in MNE, and the NIRS specific and MRI style analysis goes in MNE-NIRS. Its no extra effort for me to switch between the two repos and its had the benefit of me learning more about CIs and package management.